__tests__/app.test.js

```diff
@@ -110,7 +111,7 @@ describe('user account blog-site routes', () => {
     
     expect(res.body).toEqual({
```

Comment:
For this delete test, I think the problem is that there's a disconnect between what you're returning and what your expectation is - if the expectation you have here is what you'd like it to be, I think the steps you'll need to take are:
- switching `User.deleteById` to `UserService.delete` as the method you use in the controller
and then 
- Reworking UserService.delete to return what you're expecting in the test

Reply:
This is what I did to get it to pass, just small changes to the return value - I added in the message text because I didn't se e it elsewhere.
```js
static async delete(id) {
    const userAccount = await User.deleteById(id);
    
    const mailPreview2 = await sendEmail({
      to: userAccount.email,
      subject: 'Account Deleted!',
      html: `<h1>The Account Belonging to ${userAccount.firstName} Has Been Deleted.</h1>`
    });
    
    return {
      message: `${userAccount.firstName} blog site account has been successfully deleted.`,
      mailPreview2
    };
  }
```